client/rtvi-standard.mdx

```diff
@@ -0,0 +1,454 @@
+
+<Note>
+This page documents version 1.0 of the RTVI standard, released in June 2025.
+</Note>
+
+## Key Features
+
+<CardGroup cols={2}>
+    <Card title="Connection Management" icon="plug">
+        RTVI provides a flexible connection model that allows clients to connect to AI services and coordinate state.
+    </Card>
+    <Card title="Transcriptions" icon="text">
+    The standard includes built-in support for real-time transcription of audio streams.
+    </Card>
+    <Card title="Client-Server Messaging" icon="comments">
+    The standard defines a messaging protocol for sending and receiving messages between clients and servers, allowing for efficient communication of requests and responses.
+    </Card>
+    <Card title="Advanced LLM Interactions" icon="robot">
+    The standard supports advanced interactions with large language models (LLMs), including context management, function call handline, and search results.
+    </Card>
+    <Card title="Service-Specific Insights" icon="lightbulb">
+    RTVI supports events to provide insight into the input/output and state for typical services that exist in speech-to-speech workflows.
+    </Card>
+    <Card title="Metrics and Monitoring" icon="chart-line">
+    RTVI provides mechanisms for collecting metrics and monitoring the performance of server-side services.
+    </Card>
+</CardGroup>
+
+## Terms
+
+- **Client**: The front-end application or user interface that interacts with the RTVI server.
+- **Server**: The backend-end service that runs the AI framework and processes requests from the client.
+- **User**: The end user interacting with the client application.
+- **Bot**: The AI interacting with the user, technically an amalgamation of a large language model (LLM) and a text-to-speech (TTS) service.
+
+## RTVI Message Format
+
+The messages defined as part of the RTVI protocol adhere to the following format:
+
+```json
+{
+    "id": string,
+    "label": "rtvi-ai",
+    "type": string,
+    "data": unknown
+}
+```
+
+<ParamField path='id' type='string' required={false}>
+    A unique identifier for the message, used to correlate requests and responses.
+</ParamField>
+<ParamField path='label' type='string' required={true} default='rtvi-ai'>
+    A label that identifies this message as an RTVI message. This field is required and should always be set to `'rtvi-ai'`.
+</ParamField>
+<ParamField path='type' type='string' required={true}>
+    The type of message being sent. This field is required and should be set to one of the predefined RTVI message types listed below.
+</ParamField>
+<ParamField path='data' type='unknown' required={false}>
+    The payload of the message, which can be any data structure relevant to the message type.
+</ParamField>
+
+## RTVI Message Types
```

Comment:
Wonder if there should be a structured way of denoting who (the bot or client) is expected to send each of the messages. Having them mixed together with no labels could be a bit confusing.

Reply:
good idea... 🤔